pyseir/inference/model_fitter.py

```diff
@@ -266,9 +280,10 @@ def calculate_observation_errors(self):
         else:
             hosp_stdev = None
 
-        # Zero inflated poisson Avoid floating point errors..
-        cases_stdev[cases_stdev == 0] = 1e10
-        deaths_stdev[deaths_stdev == 0] = 1e10
+        # Zero inflated poisson Avoid floating point errors. This is set to a
+        # value that still provides some constraints toward zero.
+        cases_stdev[cases_stdev == 0] = 1e2
+        deaths_stdev[deaths_stdev == 0] = 1e2
         if hosp_stdev is not None:
             hosp_stdev[hosp_stdev == 0] = 1e10
```

Comment:
Does hosp_stdev also need to be updated?

Reply:
👍